### PR TITLE
Update Advance documents

### PR DESCRIPTION
**Pull request by chrisho**
Updated outdated settings and images while also adjusting the sequence of settings.

Related issue
https://github.com/harvester/docs/issues/373

### CODE REVIEW

File: docs/advanced/settings.md [hunk 1]
```diff
@@ -197,20 +211,41 @@ Default: `info`
...
 ```
 debug
 ```
 
+## `ntp-servers`
+
+_Available as of v1.2.0_
+
+This setting allows you to configure ntp servers for the Harvester server.
+Usually, configure the NTP servers during [installation](/dev/install/iso-install) with the default value set to `0.suse.pool.ntp.org`, or preconfigure them before initiating PXE installation.
```
Comment (guangbochen) [inline docs/advanced/settings.md:229]:
```suggestion
This setting allows you to configure NTP servers for time synchronization on the Harvester nodes.
You can define NTP servers during [installation](/install/harvester-configuration#osntp_servers) or update them later using this setting. 
```

File: docs/advanced/settings.md [hunk 2]
```diff
@@ -197,20 +211,41 @@ Default: `info`
...
+## `ntp-servers`
+
+_Available as of v1.2.0_
+
+This setting allows you to configure ntp servers for the Harvester server.
+Usually, configure the NTP servers during [installation](/dev/install/iso-install) with the default value set to `0.suse.pool.ntp.org`, or preconfigure them before initiating PXE installation. 
+
+:::caution
+
+After modifying the NTP servers, the new values will replace the previous ones for all nodes.
```
Comment (guangbochen) [inline docs/advanced/settings.md:233]:
```suggestion
Modifying the NTP servers will replace the previous values for all nodes.
```

File: docs/advanced/settings.md [hunk 3]
```diff
@@ -197,20 +211,41 @@ Default: `info`
...
+Default: ""
+
+#### Example
+
+```
+{
+  "ntpServers": [
+    "0.suse.pool.ntp.org"
+  ]
+}
```
Comment (guangbochen) [inline docs/advanced/settings.md:246]:
```suggestion
{
  "ntpServers": [
    "0.suse.pool.ntp.org",
    "1.suse.pool.ntp.org"
  ]
}
```

File: docs/advanced/settings.md [hunk 4]
```diff
@@ -342,45 +377,36 @@ Default: ""
...
 ```
 
-## `ui-plugin-index`
-
-This setting allows you to configure the JS address for the Harvester plugin (when accessing Harvester from Rancher).
-
-Default: `https://releases.rancher.com/harvester-ui/plugin/harvester-latest/harvester-latest.umd.min.js`
-
-#### Example
+## `support-bundle-namespace`
```
Comment (guangbochen) [inline docs/advanced/settings.md:395]:
```suggestion
## `support-bundle-namespaces`
```

File: docs/advanced/settings.md [hunk 5]
```diff
@@ -342,45 +377,36 @@ Default: ""
...
 
-```
-https://your.static.dashboard-ui/*.umd.min.js
-```
+_Available as of v1.2.0_
 
-## `ui-source`
+This setting allows you to configure the support bundle namespace and generate a support bundle in the `harvester-system` if the value is set to "none".
```
Comment (guangbochen) [inline docs/advanced/settings.md:399]:
```suggestion
This setting allows you to specify additional namespaces to include when collecting a support bundle. If set to "none," the support bundle will only capture resources from default system namespaces.
```

File: docs/advanced/settings.md [hunk 6]
```diff
@@ -342,45 +377,36 @@ Default: ""
...
 
-This setting allows you to configure how to load the UI source.
+Default: none
```
Comment (guangbochen) [inline docs/advanced/settings.md:401]:
```suggestion
Default: ""
```

File: docs/advanced/settings.md [hunk 7]
```diff
@@ -342,45 +377,36 @@ Default: ""
...
-The following values can be set:
+## `support-bundle-timeout`
 
-- `auto`: The default. Auto-detect whether to use bundled UI or not.
-- `external`: Use external UI source.
-- `bundled`: Use the bundled UI source.
+_Available as of v1.2.0_
 
-#### Example
+This setting allows you to  the timeout for the support bundle in minutes, use 0 to disable the timeout.
```
Comment (guangbochen) [inline docs/advanced/settings.md:407]:
```suggestion
This setting allows you to define the support bundle's default timeout duration in minutes. Use `0` to disable the timeout feature.
```

File: docs/advanced/settings.md [hunk 8]
```diff
@@ -443,3 +471,61 @@ When a host is unavailable or is powered off, the VM only reboots and does not m
...
+
+![containerd-registry](/img/v1.2/advanced/branding.png)
+
+The following options and values can be set:
+
+- Private Label: This option replaces "Harvester" with the value you provide in most places.
+- Logo: Upload light and dark logos to replace the Harvester logo in the top-level navigation header.
+- Favicon: Upload an icon to replace the Harvester favicon in the browser tab.
+- Primary Color: You can override the primary color used throughout the UI with a custom color of your choice.
+- Link Color: You can override the link color used throughout the UI with a custom color of your choice.
```
Comment (guangbochen) [inline docs/advanced/settings.md:491]:
```suggestion
- **Private Label:** This option replaces "Harvester" with the value you provide in most places.
- **Logo:** Upload light and dark logos to replace the Harvester logo in the top-level navigation header.
- **Favicon:** Upload an icon to replace the Harvester favicon in the browser tab.
- **Primary Color:** You can override the primary color used throughout the UI with a custom color of your choice.
- Link Color: You can override the link color used throughout the UI with a custom color of your choice.
```

File: docs/advanced/pcidevices.md [hunk 9]
```diff
@@ -22,6 +22,14 @@ To use the PCI devices feature, users need to enable the `pcidevices-controller`
...
 ![](/img/v1.2/vm-import-controller/EnableAddon.png)
 
+:::caution
+The `PCIDevice` is released as experimental feature in Harvester v1.1.0, the `Experimental` label is added to the addon.
+:::
+
+_Available as of v1.2.0_
+
+The `PCIDevice` is released as stabel feature in Harvester v1.2.0, the `Experimental` label is removed from the addon.
+
```
[the remark this answers is not in the document]
Comment (chrisho) [reply docs/advanced/pcidevices.md:32]: Only replace the first screenshot, I don't have an additional graphics card to capture the PCI devices list, etc.
After checking the latest UI, actually need to be updated.

**Comment by chrisho** [comment]
Need to wait until https://github.com/harvester/docs/pull/323 merges to the branch main and will rebase on it.

**Review by LucasSaintarbor — APPROVED** [review]

**Review by LucasSaintarbor — CHANGES_REQUESTED** [review]
Smal NITs.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

File: docs/advanced/settings.md [hunk 10]
```diff
@@ -342,45 +383,36 @@ Default: ""
...
-
-#### Example
-
-```
-https://your.static.dashboard-ui/*.umd.min.js
-```
+_Available as of v1.2.0_
 
-## `ui-source`
+This setting allows you to specify additional namespaces when collecting a support bundle. If set to "none," the support bundle will only capture resources from default system namespaces.
```
Comment (w13915984028) [inline docs/advanced/settings.md:405]: Add a "note" to this setting, say Harvester collects data from some pre-defined namespaces by default. ?

Comment (w13915984028) [inline docs/advanced/settings.md:405]: `from default system namespaces.` -> `from the default namespace`

system seems redundent

**Review by w13915984028 — COMMENTED** [review]
please check those 2 questions, thanks.

File: docs/advanced/settings.md [hunk 11]
```diff
@@ -36,6 +36,12 @@ Changing this setting might cause a short downtime for single-node clusters.
 This setting allows Harvester to automatically add disks that match the given glob pattern as VM storage.
 It's possible to provide multiple patterns by separating them with a comma.
 
+:::note
+
+It will only add formatted disks that are not mounted to the system.
```
Comment (LucasSaintarbor) [inline docs/advanced/settings.md:41]:
```suggestion
This setting only adds formatted disks mounted to the system.
```

File: docs/advanced/settings.md [hunk 12]
```diff
@@ -211,6 +225,33 @@ The following values can be set. The list goes from the least to most verbose lo
 debug
 ```
 
+## `ntp-servers`
+
+_Available as of v1.2.0_
+
+This setting allows you to configure NTP servers for time synchronization on the Harvester nodes.
+
+You can define NTP servers during [installation](../install/harvester-configuration#osntp_servers) or update them later using this setting.
```
Comment (LucasSaintarbor) [inline docs/advanced/settings.md:234]:
```suggestion
Using this setting, you can define NTP servers during [installation](../install/harvester-configuration#osntp_servers) or update NTP servers after installation.
```

File: docs/advanced/settings.md [hunk 13]
```diff
@@ -443,3 +477,63 @@ When a host is unavailable or is powered off, the VM only reboots and does not m
...
 }
 ```
+
+## UI Settings
+
+### `branding`
+
+_Available as of v1.2.0_
+
+This setting allows you to globally re-brand the UI by customizing the Harvester product name, logos, and color scheme. This setting allows you to globally re-brand the UI by customizing the Harvester product name, logos, and color scheme.
```
Comment (LucasSaintarbor) [inline docs/advanced/settings.md:487]:
```suggestion
This setting allows you to globally re-brand the UI by customizing the Harvester product name, logos, and color scheme.
```

**Review by LucasSaintarbor — COMMENTED** [review]
Small NITs

File: docs/advanced/settings.md (hunk 10 above)
Comment (chrisho) [reply docs/advanced/settings.md:405]: Good point, should add a note and pre-defined namespace list.

**Review by w13915984028 — APPROVED** [review]
LGTM, thanks.

**Review by LucasSaintarbor — APPROVED** [review]
LGTM, thanks

**Pull request merged**